### PR TITLE
Implement %packages --excludeWeakdeps (#1331100)

### PR DESCRIPTION
**Pull request by dashea**

### CODE REVIEW

**Comment by dashea** [comment]
I have a kickstart test, https://github.com/dashea/kickstart-tests/tree/exclude-weakdeps, which I'll turn into a real PR once all the pieces are in place. I got SUCCESS running it locally with a lovingly hand-crafted boot.iso.

File: pyanaconda/packaging/dnfpayload.py [hunk 1]
```diff
@@ -448,6 +448,12 @@ def _configure(self):
 
         conf.reposdir = REPO_DIRS
 
+        # Two reasons to turn this off:
+        # 1. Minimal installs don't want all the extras this brings in.
+        # 2. Installs aren't reproducible due to weak deps. failing silently.
```
Comment (jkonecny12) [inline pyanaconda/packaging/dnfpayload.py]: I think these are two sentences so it would be nice to capitalize the **F**ailing word.

**Comment by jkonecny12** [comment]
Other than the small issue above, this looks good to me.

**Comment by james-antill** [comment]
FWIW it's just one sentence ending (unshortened) "weak dependencies failing silently"

**Comment by dashea** [comment]
Pushed to master and f24-branch

**Pull request closed without merging**